test/integration/hotrestart_test.sh

```diff
@@ -113,6 +113,16 @@ if [[ -z "${ENVOY_IP_TEST_VERSIONS}" ]] || [[ "${ENVOY_IP_TEST_VERSIONS}" == "al
   JSON_TEST_ARRAY+=("${HOT_RESTART_JSON_V6}")
 fi
 
+# Also test for listening on UNIX domain sockets. We use IPv4 for the
+# upstreams to avoid too much wild sedding.
+HOT_RESTART_JSON_UDS="${TEST_TMPDIR}"/hot_restart_uds.json
+SOCKET_DIR="$(mktemp -d /tmp/envoy_test_hotrestart.XXXXXX)"
+cat "${TEST_RUNDIR}"/test/config/integration/server_unix_listener.json |
+  sed -e "s#{{ socket_dir }}#${SOCKET_DIR}#" | \
+  sed -e "s#{{ ip_loopback_address }}#127.0.0.1#" | \
+  cat > "${HOT_RESTART_JSON_UDS}"
+JSON_TEST_ARRAY+=("${HOT_RESTART_JSON_UDS}")
```

Comment:
Oh, I see, `hotrestart_test` is also v1 JSON. This makes me sad, but no need to fix in this PR.